src/StatsParent.js

```diff
@@ -3,8 +3,8 @@ class StatsParent {
 
   }
```

Comment:
If you don't pass anything into the constructor, you don't need to write an empty one